test/AbstractOnlineTestCase.php

```diff
@@ -23,6 +23,11 @@
 
     public static function setUpBeforeClass()
     {
+        self::$ldap = new Ldap\Ldap(static::getStandardOptions());
+    }
+
+    protected static function getStandardOptions()
+    {
```

Comment:
You should never change existing tests.That can easily introduce BC issues. Please write a new test that tests the new behaviour. The current behaviour should still be working AS-IS and that is tested by these unchanged tests…

Reply:
Looks like the reason here was because I needed the options array in my ReconnectTest class. I will duplicate this code in ReconnectTest instead.